File: Assets/Scripts/DTO/Request/Exercise/CompletedBalanceTestExerciseDTO.cs
using System.Collections.Generic;
using DTO.Request.Exercise.@base;

namespace DTO.Request.Exercise
{
    public class CompletedBalanceTestExerciseDTO : CompletedExerciseDTO
    {
        public List<Vector3DTO> phase_1;
        public List<Vector3DTO> phase_2;
        public List<Vector3DTO> phase_3;
        public List<Vector3DTO> phase_4;
    }
}

File: backend/src/main/java/vr/balance/app/DTO/request/CompletedBalanceTestExerciseDTO.java
package vr.balance.app.DTO.request;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;
import vr.balance.app.DTO.request.base.CompletedExerciseDTO;

import java.util.List;

@EqualsAndHashCode(callSuper = true)
@Data
@SuperBuilder
@NoArgsConstructor
public class CompletedBalanceTestExerciseDTO extends CompletedExerciseDTO {
    private List<Vector3> phase_1;
    private List<Vector3> phase_2;
    private List<Vector3> phase_3;
    private List<Vector3> phase_4;
}
